test/lib/buddy_check_test.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 require_relative '../test_helper'
 
-SingleCov.covered! uncovered: 4
+SingleCov.covered! uncovered: 5
```

Comment:
It's due to this line:

> JobQueue.any_instance.stubs(:delete_and_enqueue_next) # we do not properly add the job, so removal fails

As `delete_and_enqueue_next` no longer runs it won't call the `stop_expired_deploys` method.